assets/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Hallo</title>
+</head>
+<body>
+<h1>Lekker oefenen met Git</h1>
```

Comment:
Heel goed, maar misschien beter in een main element?

Reply:
:)